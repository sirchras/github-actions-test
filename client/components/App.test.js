import React from 'react'
import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import App from './App'

describe('<App />', () => {
  it('has the correct structure', () => {
    const { asFragment } = render(<App />)
    expect(asFragment()).toMatchSnapshot()
  })
  it('header has the correct text', () => {
    render(<App />)
    const header = screen.getByRole('heading', { level: 1 })
    expect(header).toHaveTextContent('React Boilerplate')
  })
})
