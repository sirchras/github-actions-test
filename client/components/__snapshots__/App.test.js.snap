// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> has the correct structure 1`] = `
<DocumentFragment>
  <h1>
    React Boilerplate
  </h1>
</DocumentFragment>
`;
